### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by KeziahF**
Hi, @BrockAllen-au can you please have a look at my code for prac 6 and comment on any improvements you think I should make? :)

### CODE REVIEW

File: prac_06/guitar.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+CURRENT_YEAR = 2022
+VINTAGE_AGE = 50
```
Comment (BrockAllen-au) [inline prac_06/guitar.py:4]: Nice use of constants.

File: prac_06/guitars.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        new_guitar = Guitar(name, year, cost)
+        guitars.append(new_guitar)
+        print(new_guitar, " added.")
+        name = input("Name: ")
+
+    print("These are my guitars:")
+    for i, guitar in enumerate(guitars, 1):
+        vintage_string = ""
+        if guitar.is_vintage():
+            vintage_string = "(vintage)"
```
Comment (BrockAllen-au) [inline prac_06/guitars.py:24]: You could make this a one liner if you wanted to:
`vintage_string = " (vintage)" if guitar.is_vintage() else ""`

File: prac_06/guitar.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class Guitar:
+    def __init__(self, name='', year=0, cost=0.0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return f'{self.name} ({self.year}) : ${self.cost:.2f}'
```
Comment (BrockAllen-au) [inline prac_06/guitar.py:16]: Will this format the cost to match the wanted output with a comma ',' separator?

File: prac_06/guitar_test.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    other = Guitar("Another Guitar", 2012, 1512.9)
+
+    print(f"{guitar.name} get_age() - Expected 100. Got {guitar.get_age()}")
+    print(f"{other.name} get_age() - Expected 10 Got {other.get_age()}")
+    print(f"{guitar.name} is_vintage - Expected True. Got {guitar.is_vintage()}")
+    print(f"{other.name} is_vintage() - Expected False. Got {other.is_vintage()}")
+
+
+
+
```
Comment (BrockAllen-au) [inline prac_06/guitar_test.py]: Is this amount of spacing in line with PEP8?

**Review by BrockAllen-au — COMMENTED** [review]
Hi @KeziahF, Enjoyed your coding. Some minor changes for you :-)

File: prac_06/guitars.py (hunk 2 above)
Comment (KeziahF) [reply prac_06/guitars.py:24]: That's a cool idea, thanks :)

File: prac_06/guitar.py (hunk 3 above)
Comment (KeziahF) [reply prac_06/guitar.py:16]: Good catch, I missed that.

File: prac_06/guitar_test.py (hunk 4 above)
Comment (KeziahF) [reply prac_06/guitar_test.py]: Another good catch, I will have a look.

**Pull request merged**